index.js

```diff
@@ -0,0 +1,53 @@
+import { DateTime } from './modules/luxon.js';
+import Book from './modules/book.js';
+import renderBooks from './modules/UI.js';
+
+const form = document.querySelector('#form');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const navLinks = document.querySelectorAll('li');
+const sections = document.querySelectorAll('section');
+
+const showSection = (id) => {
+  navLinks.forEach((li) => {
+    if (id === li.id) {
+      li.classList.add('active');
+    } else {
+      li.classList.remove('active');
+    }
+  });
+
+  sections.forEach((sec) => {
+    if (sec.classList.value !== id) {
+      sec.style.setProperty('display', 'none');
+    } else {
+      sec.style.setProperty('display', 'block');
+    }
+  });
+};
+
+renderBooks();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  Book.add(title.value, author.value);
+  title.value = '';
+  author.value = '';
+  renderBooks();
+  showSection('section-list');
+});
+
+navLinks.forEach((li) => {
+  li.addEventListener('click', () => {
+    showSection(li.id);
+  });
+});
+
+showSection('section-list');
+
+// Date
+const now = new Date();
+const date = DateTime.now().toLocaleString(DateTime.DATE_FULL);
+const time = now.toLocaleTimeString();
+document.querySelector('.date').textContent = `${date} ${time}`;
```

Comment:
- [OPTIONAL] You did great using the `luxon library` as stated in the requirement. :+1: :100: :+1: , I'll suggest you wrap this method in a `setInterval` so that the timer in the UI can be updated automatically every second. This will be great for the user experience and also give the feel of an actual digital clock. :smile: :+1:

Reply:
thank you very much for the extra feedback